feature_importance.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+import numpy as np
+import pickle
+import pandas as pd
+from sklearn.ensemble import RandomForestClassifier
+# use old version of scikit-learn==0.21.3
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MultipleLocator, PercentFormatter
+
+# Feature order for model0 to model5
+feature_names = [
```

Comment:
Use a dict instead of a list, so you can associate, which model has what features?

Reply:
Good news for this issue. The new version of sklearn has added a new attribute "feature_names_in" to store the feature names.